### PR TITLE
feat: implement bootstrap theme switcher

### PR DESCRIPTION
**Pull request by paulovareiro29**

### CODE REVIEW

File: src/partials/base/b_theme-switcher.twig [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  <ul class='dropdown-menu dropdown-menu-end' aria-labelledby='bd-theme-text'>
+    <li>
+      <button
+        type='button'
+        class='dropdown-item d-flex align-items-center active'
+        data-bs-theme-value='light'
+        aria-pressed='true'
+      >
+        <i class='bi bi-sun-fill me-2 opacity-50' data-bs-theme-icon='bi-sun-fill'></i>
+        Licht
```
Comment (koepferd) [inline src/partials/base/b_theme-switcher.twig:23]:
```suggestion
        Hell
```

File: src/partials/base/b_theme-switcher.twig [hunk 2]
```diff
@@ -0,0 +1,52 @@
+<div class='dropdown'>
+  <button
+    class='btn btn-link nav-link py-2 px-0 px-xl-2 dropdown-toggle d-flex align-items-center'
+    id='bd-theme'
+    type='button'
+    aria-expanded='false'
+    data-bs-toggle='dropdown'
+    data-bs-display='static'
+    aria-label='Umschaltthema (auto)'
```
Comment (koepferd) [inline src/partials/base/b_theme-switcher.twig:9]:
```suggestion
    aria-label='Dunkelmodus umschalten'
```

File: src/partials/base/b_theme-switcher.twig [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  >
+    <i class='bi bi-sun-fill my-1 theme-icon-active'></i>
+    <span class='d-xl-none ms-2' id='bd-theme-text'>Umschaltthema</span>
```
Comment (koepferd) [inline src/partials/base/b_theme-switcher.twig:12]:
```suggestion
    <span class='d-xl-none ms-2' id='bd-theme-text'>Dunkelmodus umschalten</span>
```

**Review by koepferd — CHANGES_REQUESTED** [review]

File: src/partials/base/b_theme-switcher.twig (hunk 2 above)
Comment (paulovareiro29) [reply src/partials/base/b_theme-switcher.twig:9]: @koepferd Should we remove the `(auto)`? It indicates the active theme.

Comment (koepferd) [reply src/partials/base/b_theme-switcher.twig]: Ah, no, we can keep the indication. Thanks :)

**Review by koepferd — APPROVED** [review]

**Pull request merged**